site/blog/2019/sep/19/helping-first-time-conference-attendees-with-welcome-wagon.rst

```diff
@@ -0,0 +1,87 @@
+Helping First-Time Conference Attendees with the Welcome Wagon
+==============================================================
+
+You walk into the room and you're wearing the *wrong* thing.
```

Comment:
I wonder if we can give this more punch by being a little more specific. For example, 

> Imagine walking into a room wearing your fanciest threads, topped with a cloche or fedora, and everyone else is wearing their Tom's and khakis. You'd probably feel mortified and find an excuse to leave quickly. If only it was like in high school when you could call a friend to ask them what they're wearing!

Reply:
Yea, I wanted to make it more specific but all the examples were often geocentric and also gendered, which I wanted to avoid.